packages/react-dev-utils/checkRequiredFiles.js

```diff
@@ -7,24 +7,51 @@
 
 'use strict';
 
-var fs = require('fs');
-var path = require('path');
-var chalk = require('chalk');
+const fs = require('fs');
+const path = require('path');
+const chalk = require('chalk');
+
+// Currently, files passed without an extension are assumed to be js-like.
+// If other types of files need to be passed later without extensions the
+// checkRequiredFiles function will need to be modified.
+const fallbackExts = [
+  '.web.js',
```

Comment:
I figure I should probably reorder this so `.js` is at the top since that's the file that will be there by default and I don't want to create unnecessary disk reads. This function just allows the run/build to continue anyway, it doesn't actually decide what file ends up resolved, just makes sure that there's at least one valid option.